src/net.rs

```diff
@@ -58,13 +58,17 @@ impl PartialEq for SignerID {
 
 impl PartialOrd for SignerID {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
```

Comment:
SignerIDのソートはPublicKeyのソートに移譲するような実装にするのがベストだと思うのですが、いかがでしょう。

Reply:
PublicKey に直接 PartialOrd を実装できないため、そのソートをコントロールすることができません。なので、ここで SignerID に実装するので良いと思っています。

Reply:
I think it can be implemented with rust-tapyrus. 
If we takes it in tapyrus-signer , the implementation here may be only #[derivative(PartialOrd, Ord)] 

- https://github.com/chaintope/rust-tapyrus/blob/master/src/util/key.rs#L80

Reply:
signerのindexを決めるのに圧縮公開鍵の辞書順を使うというのは、どちらかというと SignerID に求められる仕様のように思います。やはり、今の場所に実装するのが適切ではないですか？

Reply:
つまり、Tapyrusにおける公開鍵全般に求められる仕様ではないと思います。